ui/src/test/js/util/formatters-spec.js

```diff
@@ -17,9 +17,9 @@ describe("util/formatters-spec", function () {
         expect(formatters.memory(1)).toBe('1B');
         expect(formatters.memory(999)).toBe('999B');
         expect(formatters.memory(1999)).toBe('1.95KB');
-        expect(formatters.memory(1999999)).toBe('1.95MB');
-        expect(formatters.memory(1999999999)).toBe('1.95GB');
-        expect(formatters.memory(1999999999999)).toBe('1.95TB');
+        expect(formatters.memory(1999999)).toBe('1.91MB');
+        expect(formatters.memory(1999999999)).toBe('1.86GB');
+        expect(formatters.memory(1999999999999)).toBe('1.82TB');
```

Comment:
It's related to https://github.com/jenkinsci/pipeline-stage-view-plugin/pull/10 - for some reason the tests weren't running correctly and didn't catch the fact that we had a hard-coded dependency on (incorrect) conversion int he test specs.  This commit fixes it.